Project-2/README.md

```diff
@@ -29,9 +29,78 @@ Tip #4: Work together _as a team_.
   
 2. Use your axioms from question 1 as the basis of an ALCI T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a model of K. This may be graphical or symbolic or both.
 
+TBox:{
+PP ⊑ P 
+
+P ⊑ O 
+
+O ⊑ O¯
+
+O¯ ⊑ O
+
+D ⊑ ¬O}
+
+ABox:{
+
+(Utah, USA):P,
+
+(Bryce Canyon, Utah):PP, 
+
+(Bryce Canyon, USA):PP, 
+
+(USA, Utah):iPP, 
+
+(Utah, USA):O, 
+
+(Oregon, Utah):D,
+
+}
+
+
+ΔI = {Bryce Canyon, Utah, Oregon, USA}
+
+Named Individuals:
+
+Bryce Canyon = NP
+
+Utah = U
+
+USA = A
+
+Oregon = O
+
+
+Concept Assignments:
+
+State={U, O}
+
+Place={NP}
+
+Country={A}
+
+Role Assignments:
+P (Part_of) = {(NP, U), (U,A), (O,A), (U,U), (O,O), (A,A)
+
+PP (ProperPart_of) = {(NP,U), (NP,A), (U,A), (O,A)}
+
+iP (inverse Part_of) = {(U,NP), (A,U), (A,O), (A,NP)}
+
+iPP (inverse ProperPart_of) = {(U,NP), (A,NP), (A,U), (A,O)}
+
+O (overlaps) = {(NP,U), (NP,A), (U,A), (O,A), (NP, NP), (U,U), (A,A), (O,O)}
+
+D (disjoint) = {(O,U), (NP, O)}
+
+
+![Protege2a](https://user-images.githubusercontent.com/123985147/221180906-722d91a8-0186-4906-83d5-af76193f531f.png)
+
+
+As a Bonus here is my favorite picture I've taken at Bryce Canyon, it is at Sunrise (My favorite National Park in Utah). As well as my favorite picture I've taken in Oregon at Crater Lake (The only National Park in Oregon) simply becuase I miss the West. 
+
+![Bryce Canyon Sunrise](https://user-images.githubusercontent.com/123985147/221182562-89053251-07f0-4fa1-a50f-29f9cf168bad.jpg)
+
+![Crater Lake Wizard Island](https://user-images.githubusercontent.com/123985147/221182582-a440f7bb-c20a-4de5-b50e-8c33abafdac5.jpg)
```

Comment:
Q3. (a) My new attempt: $∃R.(∀R.(∀R¯.\top))$

Fixing x (restricted by an impicit universal quantifer), this formula says: All x is R-related to some y such that y is R-related to all z such that z is R¯-related to all x (or, all x is R-related to z).

(c) "parenthesis" is auxiliary but not logic symbol in first order logic, so a formula lacking parenthesis can still be well-formed. Also, where is the unbounded x?

(d) You are right that x is unbounded, but this formula is still well-formed. In fact, such a formula is called "open formula" in first order logic.

Reply:
Thank you regarding (a)

regarding (c) ∀y(R(x, y) → A(y))))  this is where the unbound x is. 

regarding (d) Additionally, it is my understanding of first-order that the sets of parenthesis need to have equal pairs. From my correspondence with Dr. Braun "Do you want this wff to be a conjunction? Then it is missing a final parenthesis. I think this must be what you intend, because of the occurrence of (∃y)." thus it is not well-formed

Reply:
@dmcnulty27 Oh I see the x is unbounded. Maybe a unbounded x may be an issue regarding translation, but it doesn't mean that a fomula including unbounded variables is not well-formed. In first-order logic, we have open formulas, formulas which have some unbounded variables, but they are not well-formed.

Also, I think you misunderstand the logical structure of (c). Look at (c): 

∀y(R(x, y) → ∃x(R(y, x) ∧ ∀y(R(x, y) → A(y))))

Let R(x, y) = M,  R(y,x) = N, A(y) = C, so the formula (c) can be simplified as:

∀y(M→ ∃x(N∧ ∀y(N→C)))

Then let ∀y(N→C) = S, then the formula can be further simplified as:

∀y(M→ ∃x(N∧S))

So I don't see that any parentheses is missing. "R(y, x) ∧ ∀y(R(x, y) → A(y))" is actually a conjunction of R(x,y) and a conditional  ∀y(R(x, y) → A(y)).

Reply:
@peihongx Okay so I learned that the unbound x is not unbound there is a ghost universal with the x at leat that is what John said. Although he did not use the term ghost I think implicit

Reply:
@dmcnulty27 Hi Delaney, when I said "there is an implicit universal quantifier", we should note the direction of translation: 

(1) We can translate from ∃R.C to first-order logic (FOL), we can get **∀x**∃y(Rxy & Cy). In this direction, translation requires us to add a universal quantifier which did not appear in ∃R.C before, and this is what I mean by "implicit".

(2) However, if we wanna translate from ∀x∃y(Rxy & Cy) to DL (description logic), then we have to delete a universal quantifier and thus get ∃R.C. So here I agree with you that if "∀y(R(x, y) → ∃x(R(y, x) ∧ ∀y(R(x, y) → A(y))))" does not provide a universal quantifier, then it may be dubious whether we can translate it to DL. But maybe we can do translation if we are willing to presume that there is an implicit uniervsal quantifier on the front of it. So my stance is more open.

But another key point is that **there is no missing parentheses**.

Reply:
@peihongx  FINE there is no missing parenthesis lol